frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/HypershiftClusterDetails.tsx

```diff
@@ -32,9 +32,7 @@ const HypershiftClusterDetails: React.FC = () => {
                             hostedCluster={hostedCluster}
                             nodePools={clusterNodePools}
                             clusterImages={clusterImageSets as CIM.ClusterImageSetK8sResource[]}
-                            launchToOCP={(url, newTab) =>
-                                launchToOCP(url, newTab, () => window.open(`${window.location.origin}/${url}`))
-                            }
+                            launchToOCP={(url, newTab) => launchToOCP(url, newTab)}
```

Comment:
This could be expressed as `launchToOCP={launchToOCP}`, but as I mentioned - I would probably just import the function directly in `HostedClusterProgress` unless we're actually using an alternative implementation in some case?